world/mujoco_ros/launch/mushr.launch

```diff
@@ -4,26 +4,26 @@
     <arg name="visualize" default="true"/>
     <arg name="publish_ground_truth_pose" default="false"/>
     <group ns="mushr">
-        <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>
+        <node pkg="nodelet" type="nodelet" name="estimation_standalone_nodelet"  args="manager" output="screen"/>
```

Comment:
Nit: maybe the convention we can follow is `<function>_nodelet_manager`. So this one would be `interface_nodelet_manager`.